specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2021-12-01-preview/AzureADAdministrator.json

```diff
@@ -0,0 +1,349 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MySQLManagementClient",
+    "description": "The Microsoft Azure management API provides create, read, update, and delete functionality for Azure MySQL resources including servers, databases, firewall rules and configurations.",
+    "version": "2021-12-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforMySQL/flexibleServers/{serverName}/administrators/{administratorName}": {
+      "put": {
+        "tags": [
+          "AzureADAdministrators"
+        ],
+        "operationId": "AzureADAdministrators_CreateOrUpdate",
+        "x-ms-examples": {
+          "Create an azure ad administrator": {
+            "$ref": "./examples/AzureADAdministratorCreate.json"
+          }
+        },
+        "description": "Creates or updates an existing Azure Active Directory administrator.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=69&length=2)

Consider use v3 when possible

Reply:
Since all this api-version "2021-12-01-preview" are using the same v1 version, I'd like to keep them consistence this time. Will try to change all of them to V3 in our next api-version or the stable version.